app/controllers/mobile_app/sensors_controller.rb

```diff
@@ -2,7 +2,16 @@ module MobileApp
   class SensorsController < ApplicationController
     def index
       @sensors = Sensor.order(created_at: :asc)
-      @latest_sensor_temperatures = Measurement.last_per_sensor(1).pluck(:sensor_id, :temperature).to_h
+      @latest_sensor_measurements = Measurement.last_per_sensor(1)
+                                               .pluck(:sensor_id, :temperature, :created_at)
+                                               .map { |fields| [
+                                                 fields[0],
+                                                 {
+                                                   :temperature => fields[1],
+                                                   :created_at => fields[2]&.to_time&.to_i,
+                                                 }
+                                               ]}
+                                               .to_h
```

Comment:
I'm a Ruby/rails noob, so please correct me if this is wrong style, or not the best approach 🙂